portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+
+    <div class="topnav" id="myTopnav">
+        <a href="index.html">Home</a>
+        <a href="projects.html">Projects</a>
+        <a href="contact.html">Contact</a>
+        <a href="#blog" class="active">Blog</a>
+        <a href="javascript:void(0);" class="icon" onclick="makeResponsive()">
```

Comment:
Same for this one too (indent/nest italics tag within the hyperlink tag, and likewise below) :)